contrib/basicauth/nginx/registry.conf

```diff
@@ -0,0 +1,17 @@
+# Docker registry proxy for api version 2
+
+upstream docker-registry-v2 {
+  server registryv2:5000;
+}
+
+
+server {
+  listen 5441;
```

Comment:
For this example, I think we can run on port 80. This will ensure users have nice image names, like "localhost/stevvvooe/registry".